### PR TITLE
[MIG][13.0] purchase_batch_invoicing: Migration

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc@Tecnativa TT21650
@Yajo Can you review?

### CODE REVIEW

File: purchase_batch_invoicing/tests/test_invoicing.py [hunk 1]
```diff
@@ -3,35 +3,32 @@
 from unittest import mock
 
 from odoo.exceptions import UserError
-from odoo.tests.common import SavepointCase
+from odoo.tests.common import SavepointCase, tagged
 
 mock_ns = "odoo.addons.purchase_batch_invoicing.wizards" ".purchase_batch_invoicing"
 
 
+@tagged("post_install", "-at_install")
```
Comment (yajo) [inline purchase_batch_invoicing/tests/test_invoicing.py]: If it's possible you should avoid this. Post-install tests usually break in integration environments:

```suggestion
```

**Review by yajo — CHANGES_REQUESTED** [review]
Seems good overall, just one red alert :yum:

**Review by pedrobaeza — APPROVED** [review]
Fix a little problem, but working correctly

**Comment by pedrobaeza** [comment]
As @Yajo said that is good for him, let's merge it as it's very old:

/ocabot merge nobump

**Pull request merged**